pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/ItemsRule.scala

```diff
@@ -152,27 +160,8 @@ object ItemsRule extends FieldMergeRule with MergerLogging {
     def rule(
       target: Work.Visible[Identified],
       sources: NonEmptyList[Work[Identified]]
-    ): FieldData = {
-
-      // The calm Work predicate ensures this is safe
-      val calmItem = target.data.items.head
-
-      val digitisedSources =
-        sources.filter(singleDigitalItemMetsWork or singleDigitalItemMiroWork)
-      val digitisedLocations =
-        digitisedSources.flatMap(_.data.items.flatMap(_.locations))
-
-      val sierraSources = sources.filter(sierraWork)
-      val sierraItemId =
-        sierraSources.flatMap(_.data.items.map(_.id)).headOption
-
-      List(
-        calmItem.copy(
-          locations = calmItem.locations ++ digitisedLocations,
-          id = sierraItemId.getOrElse(IdState.Unidentifiable)
-        )
-      )
-    }
+    ): FieldData =
+      sources.map(_.data.items).toList.flatten
```

Comment:
Trying to work out what the side effects of the change to how digitised locations are merged are here...

Reply:
I've done a quick test locally – if I merge Calm+Sierra+METS, it looks like the final work has two items, with the physical and digitised locations separate.

Is there an easy way to say "re-run the ItemRules" on the remaining set of works?

Reply:
Interesting - there is not an easy way to say that, but this reminds me of what I did recently here https://github.com/wellcomecollection/catalogue-pipeline/blob/74b0d98fb40492a7397ec2790f23b32525124f58/pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/ImageDataRule.scala#L19-L37

Reply:
I wonder if there should be. I'm going to open a separate ticket for it.

Reply:
It is sort of interesting to consider merging recursively until you reach the fixed point of the merging rule

Reply:
Yeah, I wonder if that's the ultimate end-state of the merger. You define a set of merging steps, and apply them all repeatedly until the result stops changing.

Reply:
Ticket here, because I don't think this is something we can solve in a pinch: https://github.com/wellcomecollection/platform/issues/5207